pulsar-testclient/src/main/java/org/apache/pulsar/testclient/PerformanceConsumer.java

```diff
@@ -438,11 +550,27 @@ public static void main(String[] args) throws Exception {
             long total = totalMessagesReceived.sum();
             double rate = messagesReceived.sumThenReset() / elapsed;
             double throughput = bytesReceived.sumThenReset() / elapsed * 8 / 1024 / 1024;
-
+            double rateAck = messageAck.sumThenReset() / elapsed;
+            long totalTxnOpSuccessNum = 0;
+            long totalTxnOpFailNum = 0;
+            double rateOpenTxn = 0;
             reportHistogram = recorder.getIntervalHistogram(reportHistogram);
 
+            if(arguments.isEnableTransaction) {
```

Comment:
please fix spaces "if(" -> "if ("
here and in the other places

Reply:
Thank you for pointing it out. This is indeed my mistake. I will correct it later.

Reply:
The problem has been solved, please help me to review again.